lib/const.rb

```diff
@@ -84,7 +84,7 @@
 
 USER = `whoami`.chomp
 
-CHROMEOS_RELEASE = File.read('/etc/lsb-release')[/CHROMEOS_RELEASE_CHROME_MILESTONE=(.+)/, 1]
+CHROMEOS_RELEASE = ( File.exist?('/etc/lsb-release') )? File.read('/etc/lsb-release')[/CHROMEOS_RELEASE_CHROME_MILESTONE=(.+)/, 1]
```

Comment:
If you are going to use an immediate if, it would be good to add `: ''` at the end.

Reply:
Done